packages/cli/lib/prompts/localDevUploadPrompt.js

```diff
@@ -0,0 +1,50 @@
+const readline = require('readline');
+const chalk = require('chalk');
+// const { promptUser } = require('./promptUtils');
+const { i18n } = require('../lang');
+
+const i18nKey = 'lib.prompts.localDevUploadPrompt';
+
+const makeUploadPrompt = message => {
```

Comment:
It may be worthwhile to add some comments in the code here, since this is breaking away from the standard prompt logic that we're using everywhere else